Formula/bcc.rb

```diff
@@ -0,0 +1,39 @@
+class Bcc < Formula
+  desc "Tools for BPF-based Linux IO analysis, networking, monitoring, and more"
+  homepage "https://github.com/iovisor/bcc"
+  url "https://github.com/iovisor/bcc/releases/download/v0.24.0/bcc-src-with-submodule.tar.gz"
+  sha256 "4ff35d71f1dc3b6fc5626bbb7644afc9eb0953be23e873c61034c8f50c589268"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "llvm" => :build
```

Comment:
What does this use in LLVM?

Reply:
I'm not too familiar with the inner workings of the project, but LLVM is required: https://github.com/iovisor/bcc/search?q=llvm

Reply:
After closer inspection, it looks like it's not just a build dep.